README.md

```diff
@@ -5,21 +5,20 @@ Delivers a human-readable diagnostic summary, written to file.
 The report is intended for development, test and production
 use, to capture and preserve information for problem determination.
 It includes JavaScript and native stack traces, heap statistics,
-platform information and resource usage etc. With the report enabled,
-reports can be triggered on unhandled exceptions, fatal errors, signals
-and calls to a JavaScript API.
+platform information and resource usage etc. Reports can be triggered on
+unhandled exceptions, fatal errors, signals and calls to JavaScript APIs.
 
-Supports Node.js v4, v6 and v7 on AIX, Linux, MacOS, SmartOS and Windows.
+Supports Node.js v4, v6 and v8 on AIX, Linux, MacOS, SmartOS and Windows.
```

Comment:
Ah yes, should have avoided that. Community is using `Node.js 8` or `version 8` now. I'll fix.